docs/questions/alerts.md

```diff
@@ -23,24 +21,35 @@ To start using alerts, an administrator will need to have set up at least one of
 
 ## Creating an alert
 
+![Get alerts](./images/get-alerts-about-this.png)
+
 To create an alert:
 
-1. Save your question
-2. Click on the sharing icon in the top-right of the screen
+1. Save your question.
+2. Click on the sharing icon in the top-right of the screen.
 3. Select **Create an alert**.
+4. Select what you want to be alerted about.  The alert options will depend on the question's visualization. See [types of alerts](#types-of-alerts).
+5. Select when you want Metabase to check the results. Metabase can check for results hourly, daily, or weekly.
+6. Select the alert's destination: [email](../configuring-metabase/email.md), [Slack](../configuring-metabase/slack.md), or to a [webhook](../configuring-metabase/webhooks.md).
 
-![Get alerts](./images/get-alerts-about-this.png)
-
-Available types of alerts will depend on the visualization you selected.
+You'll also have the option to only send the alert once. To test the alert, you can hit the **Send now** button. But just make sure that the question returns results. If the question doesn't return any results, Metabase won't send the alert.
 
 ## Types of alerts
 
-There are three kinds of things you can get alerted about in Metabase:
-
 - [Results alerts](#results-alerts): when a question returns any result.
 - [Goal line alerts](#goal-line-alerts) when a time series crosses a goal line.
 - [Progress bar alerts](#progress-bar-alerts): when a progress bar reaches or goes below its goal.
 
+## Results alerts
+
+Metabase can send you an alert on a schedule when one of your questions returns _any_ result. This kind of alert is most useful if you have a question that doesn't _usually_ return any results, but you want to know when the question _does_ return results.
```

Comment:
I always took issue with this phrasing, because I don't think this is true. Even the paragraph further on contradicts it (when we recommend an alert "see results from yesterday")

Results alert can also be used basically the same way as a scheduled dashboard subscription: send me this chart every Monday at 9 am.